desk/app/groups.hoon

```diff
@@ -1991,6 +2032,10 @@
         %section
       ?>  se-src-is-admin
       (se-c-section [section-id c-section]:c-group)
+    ::
+        %section-order
```

Comment:
Here and elsewhere we implement matching command/updates for navigation action, that used to be translated into formerly existing commands.